### ISSUE TITLE
O images not loading w/ 429 response code

### ISSUE THREAD

**Issue by 0xcadams** [livepeer/explorer#202]

**Comment by rickstaa** [comment]
Hi @0xcadams, I wanted to address the long load times for some avatars on the Explorer. Can you confirm if this was partly addressed in your commit [b34f68f](https://github.com/livepeer/explorer/commit/b34f68fd47ac2f690a8652a67cd3ce0cf8e04fb3)? Specifically, I'm referring to the changes in [[name].tsx](https://github.com/livepeer/explorer/blob/b34f68fd47ac2f690a8652a67cd3ce0cf8e04fb3/pages/api/ens-data/image/%5Bname%5D.tsx#L26-L32).

**Comment by 0xcadams** [comment]
Yes, this was addressed, and we cache image responses heavily so this should be partially resolved.

**Comment by JJassonn69** [comment]
> Yes, this was addressed, and we cache image responses heavily so this should be partially resolved.

How long is the caching window?
